app/src/main/res/layout/fragment_main.xml

```diff
@@ -80,12 +80,19 @@
         android:orientation="vertical"
         android:layout_below="@id/mainBalanceLayout">
 
-        <android.support.v7.widget.RecyclerView
-            android:id="@+id/mainTransactionList"
+        <android.support.v4.widget.SwipeRefreshLayout
+            android:id="@+id/mainTransactionListSwipe"
```

Comment:
I've used a different ID for the SwipeRefreshLayout, the old one ID is yet connected to the RecyclerView (with own adapter, etc).